infra/db/docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: "3.3"
+
+services:
+
+  db:
+    image: postgres:11-alpine
```

Comment:
alwyas try to use latest stable image:
`postgres:14-alpine`

Reply:
Will keep in mind. Thanks.